src/controllers/accounts.controller.js

```diff
@@ -27,7 +28,7 @@ exports.validate = (method) => {
         check('firstname').trim().isLength({min: 1}),
         check('lastname').trim().isLength({min: 1}),
         check('phone').trim().isNumeric().isLength({min: 10, max: 10}),
-        check('referral').trim(),
+        // check('referral').trim(),
```

Comment:
I think it should be kept?  just in case the user enters something like " ABCDEF  "

Reply:
These checks are done in the switch statement